jerry-core/ecma/base/ecma-gc.c

```diff
@@ -411,7 +411,11 @@ ecma_gc_free_native_pointer (ecma_property_t *property_p, /**< property */
     if (native_pointer_p->info_p != NULL)
     {
       ecma_external_pointer_t freecb_p = (ecma_external_pointer_t) native_pointer_p->info_p;
-      ((jerry_object_free_callback_t) freecb_p) ((uintptr_t) native_pointer_p->data_p);
+
+      if ((jerry_object_free_callback_t) freecb_p != NULL)
```

Comment:
I'd suggest splitting this out into a separate PR. This problem is not `jerry-ext/args`-specific, it's a generic issue in `jerry-core` that got revealed now (I guess). Let's fix it as soon as possible and not depend on the progress of this PR.